src/internet.ts

```diff
@@ -426,4 +438,25 @@ export class Internet {
     };
     return _password(len, memorable, pattern, prefix);
   }
+
+  /**
+   * Generates a random emoji.
+   *
+   * @param options Options object.
+   * @param options.types A list of the emoji types that should be used.
+   * @example
+   * faker.internet.emoji() // '🥰'
+   * faker.internet.emoji({types: ['food', 'nature']}) // '🥐'
```

Comment:
@ST-DDT will api-docs ignore such unformatted example docs? Or should we care about that this should be formatted?
```suggestion
   * faker.internet.emoji({ types: ['food', 'nature'] }) // '🥐'
```

Reply:
It will print them as is.
https://github.com/faker-js/faker/blob/360bec7c7f3269049eb748523f2d271b6ed36d75/scripts/apidoc/signature.ts#L128-L135
The examples are highlighted for ts though.
https://github.com/faker-js/faker/blob/360bec7c7f3269049eb748523f2d271b6ed36d75/scripts/apidoc/signature.ts#L149
We probably could run them past prettier, but then maybe as a check whether they are formatted correctly instead of just formatting them for the website.

Reply:
Oh I just recognized we don't have them only for the docs, but actually also inline docs. So we should format these in code! ^^